requirements.txt

```diff
@@ -2,4 +2,5 @@ hikari[speedups]~=2.0.0.dev101; python_version < "3.10"
 hikari~=2.0.0.dev101; python_version == "3.10"
 hikari-lightbulb~=2.0.0
 git+https://github.com/FasterSpeeding/Tanjun@c1cb6c4aa1b103cda92f6779753867102ceb12d9
+hikari-crescent>=0.3.3
```

Comment:
I would prefer if this had an upper limit. I tend to do 0.x versions as `~=0.3.3`, but if you think it'll be stable throughout 0.x, use `>=0.3.3,<1.0`. I really need to go update the others lmao. If you want to go for it, but if not no worries.

Reply:
I don't know those libs at all so I don't want to touch them